src/storage/v2/inmemory/label_property_index.cpp

```diff
@@ -931,8 +950,8 @@ auto InMemoryLabelPropertyIndex::GetAbortProcessor() const -> LabelPropertyIndex
   for (const auto &[label, per_properties] : index_) {
     for (auto const &[props, index] : per_properties) {
       for (auto const &prop : props) {
-        res.l2p[label][prop].emplace_back(&props, &index.permutations_helper);
-        res.p2l[prop][label].emplace_back(&props, &index.permutations_helper);
+        res.l2p[label][prop[0]].emplace_back(&props, &index.permutations_helper);
+        res.p2l[prop[0]][label].emplace_back(&props, &index.permutations_helper);
```

Comment:
Should these now be multimaps or exact?
This is needed for correct Abort operations

Reply:
So what is we have index :L(a.b, a.c) :L(a.c, a.b)
Hence CREATE (:L{a:{b:1, c:2}}); made two index entries, both need to be removed 

I think this is fine, just double check unit test exists.